scootapi/client/watch_job_cmd.go

```diff
@@ -76,8 +76,12 @@ func PrintJobStatus(jobStatus *scoot.JobStatus) {
 		fmt.Printf("\tTask %s {\n", taskId)
 		fmt.Printf("\t\tStatus: %s\n", taskStatus.String())
 		runStatus := jobStatus.TaskData[taskId]
-		fmt.Printf("\t\tStdout: %v\n", *runStatus.OutUri)
-		fmt.Printf("\t\tStderr: %v\n", *runStatus.ErrUri)
+		if runStatus.OutUri != nil {
```

Comment:
Is this addressing a symptom or root cause? My expectation, unless I'm forgetting what's been merged, is that these should always be set.